### ISSUE TITLE
Add-in availability 

### ISSUE THREAD

**Issue by g2k1911** [joeisapro/MonoTouch.Fabric#3]
Is there a timeframe around the availability of the add-in?  I read all the referenced material but am still having a hard time uploading my dsyms.  I don't mean to sound like I'm demanding sh*t here, I'm already happy just being able to use fabric with Xamarin.  Thanks! G2k.

**Comment by joeisapro** [comment]
It's almost ready, there's just a couple more things to iron out and then it'll be ready.  Would you be interested in testing it?

**Comment by yanbodang** [comment]
Hey, joeisapro. Is this Add-in is available for Visual Studio? Currently, I use Visual Studio to develop Xamarin Apps. And I also have some problems for uploading DSYM. So I hope I can use your add-in to handle this problems. Thanks

**Comment by joeisapro** [comment]
The initial release only supports Xamarin Studio (MAC), but support for Visual Studio is already in the pipeline.

**Comment by scordonnier** [comment]
Hi joeisapro,
I'm interested to test your add-in if it's possible.
Thanks

**Comment by joeisapro** [comment]
Hi Stephane,
The add-in should've been available weeks ago, but there's been a few breaking changes to the private Fabric REST API and some of the features (i.e.: dsym upload) no longer work.

I'm in the process of adapting the add-in but they've made it a little bit more complicated this time around.

Stay tuned.

**Comment by joeisapro** [comment]
Ok, the add-in is now available for testing.  Please let me know what version of Xamarin.Studio and Xamarin.iOS you are using as well as which kits you are interested in.

Email me at ![alt text](http://safemail.justlikeed.net/e/c00298edb985c862feae8d94dd6f68f0.png)

Thanks,
Joe.

**Comment by marioacarvalho** [comment]
@joeisapro also interested. How is that test version going?

**Comment by joeisapro** [comment]
Hi Mario,

My testing is going relatively well, but I haven't heard back from @scordonnier yet.
If you're interested in testing it, please follow the instructions outlined two messages above this one and I'll get back to you.

Thanks,
Joe.

**Comment by naturalistic** [comment]
Hi,

I have successfully used this for composing tweets, making api calls and for Tweet UI's. No issues so far, been a great help. 

Thanks,

Dan

**Comment by jzeferino** [comment]
@joeisapro what is the `Command` that we should use `After Build` to make the Crashlytcs work?

[Image Here](https://s31.postimg.org/w48f3megb/Screen_Shot_2016_06_30_at_10_16_11_AM.png)

**Comment by joeisapro** [comment]
@jzeferino Are you trying to have your dsym uploaded automatically after build?

**Comment by jzeferino** [comment]
@joeisapro  yes, but I can't achieve that via running a `command` in `After Build` like I use in Xcode.

**Comment by joeisapro** [comment]
You can use the new upload command line tool that comes with the latest
fabric Mac app. I haven't tried it myself since I use my add-in, but it
should work.

Have a look towards the bottom of this page:
https://docs.fabric.io/apple/crashlytics/missing-dsyms.html

**Comment by jzeferino** [comment]
@joeisapro Im using your plugin too. But I get can't get the command to work. What command are you passing in the `After Build`?

**Comment by jzeferino** [comment]
Im using your plugin too. But I get can't get the command to work. What command are you passing in the After Build?

**Comment by jzeferino** [comment]
@joeisapro ping.

**Comment by joeisapro** [comment]
@jzeferino Can you show me the command line you use for After Build?  The one from the link above should work.

**Comment by jzeferino** [comment]
@joeisapro Are you suggesting to paste this: `find <directory-to-search-for-dsyms> -name "*.dSYM" | xargs -I \{\} /path/to/upload-symbols -a <api-key> -p <platform> \{\}` in the  `After Build`?

**Comment by joeisapro** [comment]
@jzeferino yes, but you'll have to replace values with ones appropriate for your solution and environment.

**Comment by jzeferino** [comment]
I ended up creating a .sh script with the following code:

```
#!/bin/bash
DIRECTORY_TO_SEARCH_FOR_DSYM=$1
API_KEY=$2

echo 'Finding dSYM files and uploading to Fabric...'
find $DIRECTORY_TO_SEARCH_FOR_DSYM -name "*.dSYM" | xargs -I \{\} /Applications/Fabric.app/Contents/MacOS/upload-symbols -a API_KEY -p ios \{\}
exit 0

```

Then in the after build command I called the scrip with the correct parameters eg: `crashlytics_dsym_upload.sh ./ 31231312j3123jh2`
